aspnetcore/includes/bind7.md

```diff
@@ -13,7 +13,7 @@ Create the following `PositionOptions` class:
 
 An options class:
 
-* Must be non-abstract with a public parameterless constructor.
+* Must be non-abstract.
```

Comment:
Whilst it's true that it no longer needs a public parameterless constructor for `Bind`, it does for `Get<>`. For instance, for the abstract example in this PR, I get the following exception when binding:

```
Cannot create instance of type 'mywebapp.PositionOptions' because parameter 'age' has no matching config. 
Each parameter in the constructor that does not have a default value must have a corresponding config entry.
```

I vaguely remember writing that exception message, which is amazing considering my memory! :)